getdokan/dokan
# ⚡️Add: fontawesome library setting

### All Submissions:






### Changes proposed in this Pull Request:
Added a new setting under appearance in dokan admin dashboard called `Enable dokan fontawesome`, it enabled by default, if disabled then dokan fontawesome library will not be loaded in dokan frontend.
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/32583103/235830097-8411520d-6906-4a0b-902e-a626534336b8.png">


* Closes https://github.com/weDevsOfficial/dokan-pro/issues/2096
* Dokan pro PR: https://github.com/getdokan/dokan-pro/pull/2747
* Dokan pro PR: https://github.com/weDevsOfficial/dokan-pro/pull/2416

### How to test the changes in this Pull Request:

1. Enable and disable the setting and check if dokan fontawesome library is loaded according to setting.

### Changelog entry
> Add fontawesome library setting dokan setting appearance section


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
